src/Microsoft.ML.Transforms/RandomFourierFeaturizing.cs

```diff
@@ -36,7 +36,7 @@ namespace Microsoft.ML.Transforms
     /// This transformation is based on this paper by
     /// <a href="http://pages.cs.wisc.edu/~brecht/papers/07.rah.rec.nips.pdf">Rahimi and Recht</a>.
     /// </summary>
-    public sealed class RandomFourierExpansionTransformer : OneToOneTransformerBase
+    public sealed class RandomFourierKernelTransformer : OneToOneTransformerBase
```

Comment:
>RandomFourierKernelTransformer [](start = 24, length = 30)

I am not convinced about this change. I know what Fourier expansion is, but I have tried looking up Fourier Kernel and I have not found anything.
Also it does not seem to fit with the summary above.

Reply:
You need to search for Random Fourier Kernel Approximation.

---
In reply to: [264411188](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264411188) [](ancestors = 264411188)

Reply:
We will eename RandomFourierKernel to ApproximatedKernel everywhere.

---
In reply to: [264443488](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264443488) [](ancestors = 264443488,264411188)